l4cv/models/classifier_plus.py

```diff
@@ -0,0 +1,286 @@
+import argparse
```

Comment:
The `out_set` images are manually downloaded thumbnails from a duckduckgo search for the terms "octopus", "cuttlefish", or "nautilus" with the public domain license filter. (~2023-01-012)